src/libraries/System.Numerics.Tensors/src/CompatibilitySuppressions.xml

```diff
@@ -253,4 +253,11 @@
     <Right>lib/net9.0/System.Numerics.Tensors.dll</Right>
     <IsBaselineSuppression>true</IsBaselineSuppression>
   </Suppression>
-</Suppressions>
\ No newline at end of file
+  <Suppression>
+    <DiagnosticId>CP0021</DiagnosticId>
+    <Target>M:System.Numerics.Tensors.Tensor.StdDev``1(System.Numerics.Tensors.ReadOnlyTensorSpan{``0})``0:System.Numerics.IRootFunctions{``0}</Target>
+    <Left>lib/net9.0/System.Numerics.Tensors.dll</Left>
```

Comment:
Looks like you're missing the equivalent definition for `lib/net8.0/*` (you'll note that many suppressions are duplicated between the two)

Reply:
> What command are you passing in when building?

I tried `build.cmd -subset clr+libs+libs.pretest+packs -rc Release` but still not getting any errors locally.

> ```
> System.Numerics.Tensors.ReadOnlyTensorSpan{``0}@
> ```

I added `@` for byref now but still doesn't seem to work. The only other CP0021 supressions I can find are adding `notnull` constraint. Could it possibly be that the tool can't handle constraints that themselves are generic?

Reply:
After digging in https://github.com/dotnet/dotnet/blob/main/src/sdk/src/Compatibility/ApiCompat/Microsoft.DotNet.ApiCompatibility/Rules/CannotChangeGenericConstraints.cs I found what was missing; for type constraints, they need to be prefixed with `T:`